### ISSUE TITLE
Add session_affinity variable to serverless_neg module.

### ISSUE THREAD

**Issue by uap-universe** [terraform-google-modules/terraform-google-lb-http#274]
### TL;DR

The other module already has the possibility to configure session_affinity. Please add this to the serverless_neg module as well.

### Terraform Resources

https://github.com/terraform-google-modules/terraform-google-lb-http/blob/c5cd0d0f2210cd50581f44fafa1995f512f7d997/modules/serverless_negs/main.tf#L169-L212

### Detailed design

Add a variable `session_affinity: string` to variables.tf and then the following line to the resource:

`session_affinity                      = lookup(each.value, "session_affinity", "NONE")`

### Additional information

To have a more complete solution, other variables should also be made configurable, for example `affinity_cookie_ttl_sec`.

**Comment by uap-universe** [comment]
This would still be very appreciated.

**Comment by uap-universe** [comment]
I still think this feature should be implemented. I would do it myself and create a PR, but I don't see how I could create the necessary testing environment.
